.github/ISSUE_TEMPLATE/inception-request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Inception request
+about: Suggest an idea for OFN's Inception process
+title: "[Inception Request: Name your problem statement here]"
+labels: 'Blocked:Design Input needed, Blocked: Tech Input needed, Blocked:Product
+  input neeed'
```

Comment:
I think at the beginning of the Inception process an issue (or Inception request) is not **blocked** but rather _in need_ for input. So I would exchange or add for "Needs Design Work" etc,

Reply:
Change this in the template but this was originally in our labels text so I edited it there too :)